tools/PI/DevHome.PI/Helpers/ExternalTool.cs

```diff
@@ -180,29 +216,32 @@ internal async override void InvokeTool(Window? parentWindow, int? targetProcess
                     {
                         // Note: running most msc files requires elevation.

```

Comment:
I would expect to see a check for running as admin next to this comment.

Can a check be added to PI  to check this when we know the user wants to run mmc?

Reply:
It's actually mmc.exe that will drive the elevation in this case. PI doesn't need to worry about it.... when we go to launch the .msc file, if MMC thinks it needs to run elevated, it will pop the UAC.